ocs_ci/ocs/ui/block_pool.py

```diff
@@ -21,6 +22,7 @@ class BlockPoolUI(PageNavigator):
     def __init__(self, driver):
         super().__init__(driver)
         ocp_version = get_ocp_version()
+        self.ocp_version_full = version.get_semantic_ocp_version_from_config()
```

Comment:
you can use this var.
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/ui/base_ui.py#L384

Reply:
Updated.